convert2rhel/unit_tests/conftest.py

```diff
@@ -60,6 +66,25 @@ def setup_logger(tmpdir):
     setup_logger_handler(log_name="convert2rhel", log_dir=str(tmpdir))
 
 
+@pytest.fixture
+def system_cert_with_target_path(monkeypatch, tmpdir, request):
+
+    mark = request.node.get_closest_marker("cert_filename")
+    if not mark:
+        temporary_filename = "filename"
+    else:
+        temporary_filename = mark.args[0]
+
+    tmp_file = tmpdir / temporary_filename
+
+    monkeypatch.setattr(cert.SystemCert, "_get_cert", value=mock.Mock(return_value=("anything", "anything")))
+    monkeypatch.setattr(cert.SystemCert, "_get_target_cert_path", value=mock.Mock(return_value=str(tmp_file)))
+
+    sys_cert = cert.SystemCert()
+
+    return sys_cert
```

Comment:
I'm just curious about this fixture and why we are going the route to create a new marker instead of using a parametrized fixture here... Don't get me wrong, I think this is cool and I learned something different today... 

If using a marker is indeed the easiest option here (Or actually, the better option) what do you think about making a little comment above the line `72` regarding the use of `pytest.mark`? I believe not many people will take a look at this and guess what's going on on the fly. Also, could be good to point out the [pytest docs](https://docs.pytest.org/en/7.1.x/example/markers.html) to reference what a custom marker is, this way you don't need to type in all of the information by hand. 

_Note_: I can see the point here about using a custom marker to make it easier in the tests by just calling a `@pytest.mark.cert_filename` and it's done, but also, I think that a parametrized fixture would be simpler to use. Anyway, just curious about this one!

Reply:
Hey @r0x0d I'm not exactly sure why it is a better reason to use the marker over using paramertrized, @abadger helped me out with this code and I think he could supply a better answer then I can.

Reply:
I like to limit use of parametrize to just things that need to run the test multiple times with different data each time.  For constant data, I pass the data another way (and it looks like mark is the way that pytest recommends).

The problem with using parametrize for constant data is that making sure that parametrize's data covers every permutation of the data is difficult.  Throwing a constant into that will leave people wondering whether the constant should vary just like the other pieces of data.  IMHO, it's better to move this information into something other than the parametrize data.